src/lib.rs

```diff
@@ -78,3 +78,88 @@ pub use self::db::{Database, Error};
 #[cfg(feature = "jemalloc")]
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
+
+use std::path::PathBuf;
+
+/// Python Extension
+use pyo3::{prelude::*, wrap_pyfunction};
+use storage::SecondaryStorageOptions;
+/// Open a database for user, user can specify the path of database file
+#[pyfunction]
+fn open(path: String) -> PyResult<PythonDatabase> {
```

Comment:
Could we move these code into a new file?

Reply:
I don't know. Let me have a try.